IterateSDK/Helpers/Color.swift

```diff
@@ -27,3 +27,7 @@ extension UIColor {
         self.init(red:red, green:green, blue:blue, alpha:1)
     }
 }
+
+enum Colors: String {
+    case LightBlack = "#1f1f1f"
```

Comment:
What's the reason for this hard coded value? It's possible we do this elsewhere and that's fine, but jsut checking...